cmd/tendermint/commands/run_node.go

```diff
@@ -79,8 +81,45 @@ func runNode(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	moduleToLogLevelMap, err := tmflags.ParseLogLevel(config.LogLevel)
+	if err != nil {
+		return err
+	}
+
+	options := make([]log.Option, 0)
+	for module, level := range moduleToLogLevelMap {
+		if module == "*" {
+			switch level {
+			case "debug":
+				options = append(options, log.AllowDebug())
```

Comment:
can't we do these as `AllowXxxWith` with no args ? and then only need the switch statement once ?